<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Moteur_De_Recherche</class>
 <widget class="QMainWindow" name="Moteur_De_Recherche">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="pushButton_2">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>410</y>
      <width>91</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Requete</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_3">
    <property name="geometry">
     <rect>
      <x>500</x>
      <y>410</y>
      <width>80</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>Calcul R/P</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_4">
    <property name="geometry">
     <rect>
      <x>670</x>
      <y>410</y>
      <width>80</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>PushButton</string>
    </property>
   </widget>
   <widget class="QLabel" name="ImShow">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>70</y>
      <width>201</width>
      <height>261</height>
     </rect>
    </property>
    <property name="text">
     <string>Ouvrir</string>
    </property>
   </widget>
   <widget class="QPushButton" name="open_btn">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>370</y>
      <width>80</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>Open</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>410</y>
      <width>80</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>PushButton</string>
    </property>
   </widget>
   <widget class="QProgressBar" name="progressBar">
    <property name="geometry">
     <rect>
      <x>167</x>
      <y>450</y>
      <width>471</width>
      <height>23</height>
     </rect>
    </property>
    <property name="value">
     <number>12</number>
    </property>
   </widget>
   <widget class="QComboBox" name="comboBox">
    <property name="geometry">
     <rect>
      <x>660</x>
      <y>50</y>
      <width>101</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="Search">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>90</y>
      <width>221</width>
      <height>241</height>
     </rect>
    </property>
    <property name="text">
     <string>Moteur de recherche</string>
    </property>
   </widget>
   <widget class="QLabel" name="Calcul">
    <property name="geometry">
     <rect>
      <x>580</x>
      <y>113</y>
      <width>181</width>
      <height>221</height>
     </rect>
    </property>
    <property name="text">
     <string>Calcul R/P</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>19</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
